doc/api/fs.md

```diff
@@ -7558,12 +7564,12 @@ of one before invoking the other:
 ```mjs
 import { rename, stat } from 'node:fs/promises';
 
-const from = '/tmp/hello';
-const to = '/tmp/world';
+const fromDir = '/tmp/hello';
+const toDir = '/tmp/world';
```

Comment:
Does this imply that `/tmp/hello` and `/tmp/world` are directories? I think, something like
```suggestion
const fromPath = '/tmp/hello';
const toPath = '/tmp/world';
```
or
```suggestion
const oldPath = '/tmp/hello';
const newPath = '/tmp/world';
```
would be less confusing.

Reply:
I agree 👍 thanks!